metrics/metricskey/constants_eventing.go

```diff
@@ -59,16 +59,16 @@ var (
 		LabelLocation,
 		LabelClusterName,
 		LabelNamespaceName,
-		LabelName,
```

Comment:
Is `LabelName` still used somewhere? Could we remove its definition?

Reply:
I'll rollback the change on sources. If I do so, then nothing should break in other repos, and will make it easy for cherry-picking to 0.10.
So, yeah, this Name will be used.